cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -348,7 +348,23 @@ if(PARQUET_BUILD_BENCHMARKS)
 endif()
 
 ## Apache Arrow
-find_package(Arrow)
+pkg_check_modules(ARROW arrow)
+pkg_check_modules(ARROW_IO arrow-io)
+if (ARROW_FOUND AND ARROW_IO_FOUND)
+  set(ARROW_INCLUDE_DIR ${ARROW_INCLUDE_DIRS})
+  set(ARROW_LIBS ${ARROW_LIBRARY_DIRS})
+
+  set(ARROW_LIB_NAME libarrow)
+  set(ARROW_IO_LIB_NAME libarrow_io)
```

Comment:
Is there a way to make this work for MSVC (no lib prefix)? We aren't supporting visual studio yet, but hopefully soon